.github/ISSUE_TEMPLATE/release-planning.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Release Planning
+about: File a release planning issue
+title: '[RELEASE] <title>'
+labels: kind/release
+---
+
+### Version
+<!--
+The version number follows the semantic versioning scheme.
+- Major version is incremented for any backwards incompatible changes.
+- Minor version is incremented for new features, but not backwards incompatible changes.
+- Patch version is incremented for bug fixes.
+-->
+
+### Important Dates
+<!--
+The release date is the date of the release.
+The code freeze date is the date of the code freeze.
+It is typically 2 weeks before the release date.
+-->
+
+**Release Date:**
+
+**Code Freeze Date:**
+
+### Major Changes
+<!--
+Please list the major changes that are planned for this release.
+This should include new features, bug fixes, and any other changes that are planned.
+-->
+
+### Checklist
+
+- [ ] Feature Complete
+- [ ] Code Frozen
+- [ ] (Optional) Release Candidate Tagged
```

Comment:
can this be optional ? 

Shouldn't we actually run some automated tests on RCs to "prove" their stable enough to be considered ?